tests/format/js/conditional/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -312,27 +318,28 @@ const typeofExample =
     definition.encode(
       typeof row[field] !== "undefined" ? row[field]
       : typeof definition.default !== "undefined" ? definition.default
-      : null
+      : null,
     )
   : typeof row[field] !== "undefined" ? row[field]
   : typeof definition.default !== "undefined" ? definition.default
-  : null
+  : null;
```

Comment:
Was there a bug previously where it didn't add a `;` at the end of the statement?

Reply:
nope this is the input